src/java/picard/vcf/GenotypeConcordance.java

```diff
@@ -62,14 +62,69 @@
  * @author George Grant
  */
 @CommandLineProgramProperties(
-        usage = "Calculates the concordance between genotype data for two samples in two different VCFs - one being considered the truth (or reference) " +
-                "the other being considered the call.  The concordance is broken into separate results sections for SNPs and indels.  Summary and detailed statistics are reported\n\n" +
-                "Note that for any pair of variants to compare, only the alleles for the samples under interrogation are considered " +
-                "and MNP, Symbolic, and Mixed classes of variants are not included.",
-        usageShort = "Calculates the concordance between genotype data for two samples in two different VCFs",
+        usage = GenotypeConcordance.USAGE_SUMMARY + GenotypeConcordance.USAGE_DETAILS,
+        usageShort =  GenotypeConcordance.USAGE_SUMMARY,
         programGroup = VcfOrBcf.class
 )
 public class GenotypeConcordance extends CommandLineProgram {
+    static final String USAGE_SUMMARY = "Calculates the concordance between genotype data for samples in two different" +
+            "VCFs, your callset and a standard or 'truthset'.";
+    static final String USAGE_DETAILS = "Genotype concordance is the fraction of correct assigned genotypes divided by the" +
+            " total number of true-positive (TP) variant alleles.  Genotype concordance is calculated by dividing the number of validated " +
+            "genotypes in a call set by the total number of TP alleles.  Validated genotypes are variant " +
+            "genotypes that match a gold-standard data set.  The gold-standard data set can be any curated data set " +
```

Comment:
I'm not comfortable with this use of the term "gold-standard". It's an oft-abused term. Let's just use "truth set". Also, the description is a little too constrained. Generally speaking, a truth set can be any set of variants that you decide represents the closest possible approximation of biological truth. Typically, the best truth sets are those that have been cross-validated with multiple technologies. But this is not always available, and sometimes people have to make do with bootstrapped truth sets. It's important to reflect that as a valid if imperfect option here.